flaml/integrations/oai/completion.py

```diff
@@ -481,6 +463,7 @@ def tune(
         """Tune the parameters for the OpenAI API call.
 
         TODO: support parallel tuning with ray or spark.
+        TODO: support agg_method as in test
```

Comment:
```suggestion
        TODO: support agg_method as in tune
```

Reply:
will fix later.

Reply:
"as in test" is correct because you have added the support in test.